e2e-tests/insurance/cypress/e2e/journeys/application-submission/multiple-policy-type/business-conditions/submit-an-application-multiple-policy-type-business-fully-populated.spec.js

```diff
@@ -26,4 +26,8 @@ context('Insurance - submit an application - Multiple policy - fully populated b
   it('should successfully submit the application and redirect to `application submitted`', () => {
     cy.assertApplicationSubmittedUrl(referenceNumber);
   });
+
+  it('should render as expected in the dashboard', () => {
```

Comment:
should we add render as submitted or similar for these its? just so its more clear

Reply:
Good shout, i've updated it to this:

```
it('should render in a `submitted` state in the dashboard', () => ...
```